src/main/java/IonErm/utility/DogNameGenerator.java

```diff
@@ -0,0 +1,28 @@
+package IonErm.utility;
+
+import java.util.Random;
+
+public class DogNameGenerator {
```

Comment:
All this can be changed with RandomStringUtils.randomAlphanumeric(17).toUpperCase();